buildlib/test_pr.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -eExl
```

Comment:
IMO this script is better to implement inside Azure pipeline as a `bash` step. Currently, the real test run is skipped, but we don't see it. You can use `##vso[task.complete result=Skipped;]No IB devices found` for such cases. More about Azure logging commands [here](https://docs.microsoft.com/en-us/azure/devops/pipelines/scripts/logging-commands?view=azure-devops&tabs=bash).